diff --git a/clients/python/src/mr_openapi/configuration.py b/clients/python/src/mr_openapi/configuration.py
index 795d09e8..bab01d69 100644
--- a/clients/python/src/mr_openapi/configuration.py
+++ b/clients/python/src/mr_openapi/configuration.py
@@ -17,7 +17,6 @@ import logging
 from logging import FileHandler
 import sys
 from typing import Optional
-import urllib3
 
 import http.client as httplib
 
@@ -110,7 +109,6 @@ class Configuration:
         """Logging Settings
         """
         self.logger["package_logger"] = logging.getLogger("mr_openapi")
-        self.logger["urllib3_logger"] = logging.getLogger("urllib3")
         self.logger_format = '%(asctime)s %(levelname)s %(message)s'
         """Log format
         """
@@ -335,21 +333,6 @@ class Configuration:
             else:
                 return key
 
-    def get_basic_auth_token(self):
-        """Gets HTTP basic authentication header (string).
-
-        :return: The token for basic HTTP authentication.
-        """
-        username = ""
-        if self.username is not None:
-            username = self.username
-        password = ""
-        if self.password is not None:
-            password = self.password
-        return urllib3.util.make_headers(
-            basic_auth=username + ':' + password
-        ).get('authorization')
-
     def auth_settings(self):
         """Gets Auth Settings dict for api client.
 
