diff --git a/clients/python/src/mr_openapi/configuration.py b/clients/python/src/mr_openapi/configuration.py
index 50b0c1f2..068de622 100644
--- a/clients/python/src/mr_openapi/configuration.py
+++ b/clients/python/src/mr_openapi/configuration.py
@@ -21,8 +21,6 @@ import sys
 from typing import Any, ClassVar, Dict, List, Literal, Optional, TypedDict, Union
 from typing_extensions import NotRequired, Self
 
-import urllib3
-
 
 JSON_SCHEMA_VALIDATION_KEYWORDS = {
     'multipleOf', 'maximum', 'exclusiveMaximum',
@@ -238,7 +236,6 @@ class Configuration:
         """Logging Settings
         """
         self.logger["package_logger"] = logging.getLogger("mr_openapi")
-        self.logger["urllib3_logger"] = logging.getLogger("urllib3")
         self.logger_format = '%(asctime)s %(levelname)s %(message)s'
         """Log format
         """
@@ -472,21 +469,6 @@ class Configuration:

         return None
 
-    def get_basic_auth_token(self) -> Optional[str]:
-        """Gets HTTP basic authentication header (string).
-
-        :return: The token for basic HTTP authentication.
-        """
-        username = ""
-        if self.username is not None:
-            username = self.username
-        password = ""
-        if self.password is not None:
-            password = self.password
-        return urllib3.util.make_headers(
-            basic_auth=username + ':' + password
-        ).get('authorization')
-
     def auth_settings(self)-> AuthSettings:
         """Gets Auth Settings dict for api client.
 
